src/LinuxTracing/SwitchesStatesNamesVisitor.cpp

```diff
@@ -173,14 +175,27 @@ void SwitchesStatesNamesVisitor::Visit(uint64_t event_timestamp,
 }
 
 void SwitchesStatesNamesVisitor::Visit(uint64_t event_timestamp,
-                                       const SchedWakeupPerfEventData& event_data) {
+                                       const SchedSwitchPerfEventData& event_data) {
+  VisitSchedSwitch(event_timestamp, event_data, /*wait_for_callstack=*/false);
```

Comment:
Also here regarding `wait_for_callstack`.

Reply:
Done, here and elsewere.